### DISCUSSION TITLE
Z2m only starts when folder snippets is created

### DISCUSSION THREAD

**Discussion by dawiinci** [zwave-js/zwave-js-ui#2623] [General]
I just installed the latest version 7.0.0 using an existing store folder. It failed to start and stated that the folder snippets was missing. I created this folder (empty):
.../zwavejs2mqtt/server/snippets

Now it works again. Is this a bug or am I missing something?

**Comment by robertsLando** [comment]
@dawiinci I think you are using master version, anyway I just fixed this problem [here](https://github.com/zwave-js/zwavejs2mqtt/commit/c201cddebd7a1a37575142463dfa9bc72c8e9e1f)

**Reply by dawiinci** [reply]
I am. Shouldn't I install master?

Thanks for fixing.

**Reply by robertsLando** [reply]
In order to be 99% sure there are no unexpected issues `latest` tag would be better, BTW master should be good too but less tested
